lib/init.js

```diff
@@ -79,6 +79,6 @@ module.exports = function (opts, callback) {
       );
     }
 
-    callback(err);
+    callback(err, results);
```

Comment:
Let's revert for now and update the doc string. It's internal API that is unused.